packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -769,7 +769,7 @@ class SortedTable extends Component {
     )
 
     const dropdownItemsPerPage = (
-      <DropdownButton bsStyle='info' title={itemsPerPage}>
+      <DropdownButton bsStyle='info' id='itemsPerPage' title={itemsPerPage}>
```

Comment:
If there are 2 `SortedTable`s on the same view, then there will be a duplicated `id`.

Reply:
Should we pass to `SortedTable` a required props to defined the `dropdown id` ?

Reply:
No, it would be too annoying to use, IMO. I think we can either use the `stateUrlParam` prop or generate a random ID.